photon-avro-schemas/src/main/avro/BayesianLinearModelAvro.avsc

```diff
@@ -9,6 +9,12 @@
             "type": "string"
         },
         {
+            "default": null,
+            "name": "modelClass",
+            "type": "string",
+            "doc": "The fully-qualified class name of enclosing GLM model class. E.g.: com.linkedin.photon.ml.supervised.classification.LogisticRegressionModel"
+        },
+        {
             "name": "means",
             "type": {
                 "items": "NameTermValueAvro",
```

Comment:
@XianXing Please have a look at this commit. This adds the model classname to the serialized model, so that we can rehydrate the right class when loading the model later for scoring.